src/tests/ftest/io/io_aggregation.yaml

```diff
@@ -16,19 +18,19 @@ server_config:
       - FI_SOCKETS_CONN_TIMEOUT=2000
       - DD_MASK=all
     bdev_class: nvme
-    bdev_list: ["0000:81:00.0"]
+    bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
 pool:
   createmode:
     mode_RW:
       mode: 146
   createset:
     setname: daos_server
   createsize:
-    scm_size: 5000000000
-    nvme_size: 10000000000
+    size: 45000000000
```

Comment:
Better to use pool % so it can work on smaller cluster

Reply:
Used in dev environment:
scm_size: 20%
nvme_size: 50%

worked fine

Reply:
Should be resolved